### ISSUE TITLE
Credit Notes

### ISSUE THREAD

**Issue by jstrnbrg** [shiftdevices/saleor#21]
- [ ] Ability to manually create orders with negative value

- [ ] Automatically create voucher of the same value 

- [ ] Instead of creating an invoice for that order create a credit note, put voucher code on credit note

**Comment by elvissio** [comment]
Ability to manually create orders with negative value 
---as an admin in the name of selected user. Users should not be able to create negative orders.

  Automatically create voucher of the same value
---why? 

  Automatically update inventory accordingly (same popup that asks which warehouse the products will be sent to)
---OK

  Instead of creating an invoice for that order create a credit note, put voucher code on credit note
- why is voucher needed?
and
Credit note requires just a different report named Credit note and alternative texts as CN is just negative Invoice.

**Comment by elvissio** [comment]
> How do you want to manage the credit notes? We want them to have free credit for future orders. Doing that via vouchers, (again I know) is the easist way to do it. Alternatively we would need to keep track of each customers available balance which is something saleor currently does not do.<
Yes, that's why ERP is a must. It might be that the easiest _for now_ would be vouchers, but that doesn't mean it is easy. I'm scared now :)
